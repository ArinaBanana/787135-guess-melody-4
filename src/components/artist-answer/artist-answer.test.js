import React from "react";
import renderer from "react-test-renderer";
import ArtistAnswer from "./artist-answer.jsx";

const answer = {
  picture: `path`,
  artist: `John Snow`
};

it(`Should render Artist Answer`, () => {
  const tree = renderer.create(<ArtistAnswer index={1} isChecked={true} answer={answer} onChangeAnswer={() => {}} />).toJSON();

  expect(tree).toMatchSnapshot();
});
