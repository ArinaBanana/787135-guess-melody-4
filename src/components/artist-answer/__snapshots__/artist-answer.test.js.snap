// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Artist Answer 1`] = `
<div
  className="artist"
>
  <input
    checked={true}
    className="artist__input visually-hidden"
    id="answer-1"
    name={
      Object {
        "artist": "John Snow",
        "picture": "path",
      }
    }
    onChange={[Function]}
    type="radio"
    value="answer-1"
  />
  <label
    className="artist__name"
    htmlFor="answer-1"
  >
    <img
      alt="John Snow"
      className="artist__picture"
      src="path"
    />
    John Snow
  </label>
</div>
`;
